### ISSUE TITLE
Change locale with Screengrab make Pixel Launcher crash on Android 25

### ISSUE THREAD

**Issue by mathroule** [fastlane/fastlane#10418]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description

When running Screengrab to capture screenshots on an AVD (Nexus 5X Android 7.1.1 Google APIs), the locale change make Pixel Launcher app crash (package com.google.android.apps.nexuslauncher). Unfortunatly a dialog appear to inform the user of the crash, like this: 
![screenshot_1506435146](https://user-images.githubusercontent.com/4117425/30865321-8b3d3432-a2d6-11e7-8325-e1c76771e259.png)

If I disable LocaleTestRule, everything runs fine, except locales are not changed of course.

If I run this command `adb shell pm clear com.google.android.apps.nexuslauncher` during my test (i.e when dialog is displayed), the dialog is dismissed, but this workaround is not convenient at all.

AVD stack trace of com.google.android.apps.nexuslauncher crash:
```
com.google.android.apps.nexuslauncher E/AndroidRuntime: FATAL EXCEPTION: main
	 Process: com.google.android.apps.nexuslauncher, PID: 2296
	 java.lang.NullPointerException: Attempt to invoke virtual method 'android.view.ViewParent android.view.View.getParent()' on a null object reference
		 at com.android.launcher3.keyboard.ViewGroupFocusHelper.computeLocationRelativeToContainer(SourceFile:54)
		 at com.android.launcher3.keyboard.ViewGroupFocusHelper.computeLocationRelativeToContainer(SourceFile:64)
		 at com.android.launcher3.keyboard.ViewGroupFocusHelper.computeLocationRelativeToContainer(SourceFile:64)
		 at com.android.launcher3.keyboard.ViewGroupFocusHelper.computeLocationRelativeToContainer(SourceFile:64)
		 at com.android.launcher3.keyboard.ViewGroupFocusHelper.viewToRect(SourceFile:42)
		 at com.android.launcher3.keyboard.FocusIndicatorHelper.getDrawRect(SourceFile:147)
		 at com.android.launcher3.keyboard.FocusIndicatorHelper.invalidateDirty(SourceFile:128)
		 at com.android.launcher3.keyboard.FocusIndicatorHelper.onFocusChange(SourceFile:190)
		 at android.view.View.onFocusChanged(View.java:6248)
		 at android.widget.TextView.onFocusChanged(TextView.java:8401)
		 at android.view.View.handleFocusGainInternal(View.java:5965)
		 at android.view.View.requestFocusNoSearch(View.java:9130)
		 at android.view.View.requestFocus(View.java:9109)
		 at android.view.ViewGroup.onRequestFocusInDescendants(ViewGroup.java:2875)
		 at android.view.ViewGroup.requestFocus(ViewGroup.java:2831)
		 at android.view.ViewGroup.onRequestFocusInDescendants(ViewGroup.java:2875)
		 at android.view.ViewGroup.requestFocus(ViewGroup.java:2831)
		 at android.view.ViewGroup.onRequestFocusInDescendants(ViewGroup.java:2875)
		 at android.view.ViewGroup.requestFocus(ViewGroup.java:2831)
		 at android.view.ViewGroup.onRequestFocusInDescendants(ViewGroup.java:2875)
		 at com.android.launcher3.dragndrop.DragLayer.onRequestFocusInDescendants(SourceFile:1084)
		 at android.view.ViewGroup.requestFocus(ViewGroup.java:2831)
		 at android.view.ViewGroup.onRequestFocusInDescendants(ViewGroup.java:2875)
		 at android.view.ViewGroup.requestFocus(ViewGroup.java:2831)
		 at android.view.ViewGroup.onRequestFocusInDescendants(ViewGroup.java:2875)
		 at android.view.ViewGroup.requestFocus(ViewGroup.java:2831)
		 at android.view.ViewGroup.onRequestFocusInDescendants(ViewGroup.java:2875)
		 at android.view.ViewGroup.requestFocus(ViewGroup.java:2831)
		 at android.view.ViewGroup.onRequestFocusInDescendants(ViewGroup.java:2875)
		 at android.view.ViewGroup.requestFocus(ViewGroup.java:2834)
		 at android.view.View.requestFocus(View.java:9076)
		 at android.view.View.requestFocus(View.java:9055)
		 at android.view.View.rootViewRequestFocus(View.java:6147)
		 at android.view.ViewGroup.removeAllViewsInLayout(ViewGroup.java:4925)
		 at com.android.launcher3.PagedView.removeAllViewsInLayout(SourceFile:983)
		 at android.view.ViewGroup.removeAllViews(ViewGroup.java:4856)
		 at com.android.launcher3.Workspace.removeAllWorkspaceScreens(SourceFile:677)
		 at com.android.launcher3.Launcher.startBinding(SourceFile:3559)
		 at com.android.launcher3.LauncherModel$LoaderTask$7.run(SourceFile:2575)
		 at com.android.launcher3.DeferredHandler$Impl.handleMessage(SourceFile:49)
		 at android.os.Handler.dispatchMessage(Handler.java:102)
		 at android.os.Looper.loop(Looper.java:154)
		 at android.app.ActivityThread.main(ActivityThread.java:6119)
		 at java.lang.reflect.Method.invoke(Native Method)
		 at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:886)
		 at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:776)
```

### Environment

```
### Stack

| Key                         | Value                                                          |
| --------------------------- | -------------------------------------------------------------- |
| OS                          | debian_based stretch/sid                                       |
| Ruby                        | 2.3.1                                                          |
| Bundler?                    | false                                                          |
| Git                         | git version 2.7.4                                              |
| Installation Source         | /usr/local/bin/fastlane                                        |
| Host                        | Ubuntu 16.04.3 LTS debian_based stretch/sid (4.4.0-96-generic) |
| Ruby Lib Dir                | /usr/lib                                                       |
| OpenSSL Version             | OpenSSL 1.0.2g  1 Mar 2016                                     |
| Is contained                | false                                                          |
| Is homebrew                 | false                                                          |
| Is installed via Fabric.app | false                                                          |


### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   |             |   |
| LANGUAGE | en          | 🚫 |


### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.59.0  | ✅ Up-To-Date  |


### Loaded fastlane plugins:

| Plugin                                      | Version | Update-Status |
| ------------------------------------------- | ------- | ------------- |
| fastlane-plugin-automated_test_emulator_run | 1.5.1   | ✅ Up-To-Date  |
| fastlane-plugin-changelog                   | 0.7.0   | ✅ Up-To-Date  |
| fastlane-plugin-demo_mode                   | 1.0.1   | ✅ Up-To-Date  |


### Loaded gems:

| Gem                                         | Version      |
| ------------------------------------------- | ------------ |
| slack-notifier                              | 1.5.1        |
| rouge                                       | 2.0.7        |
| xcpretty                                    | 0.2.8        |
| terminal-table                              | 1.8.0        |
| CFPropertyList                              | 2.3.5        |
| multipart-post                              | 2.0.0        |
| word_wrap                                   | 1.0.0        |
| public_suffix                               | 2.0.5        |
| tty-screen                                  | 0.5.0        |
| babosa                                      | 1.0.2        |
| colored                                     | 1.2          |
| highline                                    | 1.7.8        |
| commander-fastlane                          | 4.4.5        |
| faraday-cookie_jar                          | 0.0.6        |
| fastimage                                   | 2.1.0        |
| gh_inspector                                | 1.0.3        |
| json                                        | 1.8.3        |
| mini_magick                                 | 4.5.1        |
| multi_xml                                   | 0.6.0        |
| rubyzip                                     | 1.2.1        |
| security                                    | 0.1.3        |
| xcpretty-travis-formatter                   | 0.0.4        |
| declarative                                 | 0.0.10       |
| declarative-option                          | 0.1.0        |
| representable                               | 3.0.4        |
| addressable                                 | 2.5.2        |
| mime-types-data                             | 3.2016.0521  |
| mime-types                                  | 3.1          |
| httpclient                                  | 2.8.3        |
| google-api-client                           | 0.13.6       |
| io-console                                  | 0.4.5        |
| excon                                       | 0.59.0       |
| plist                                       | 3.3.0        |
| faraday                                     | 0.13.1       |
| jwt                                         | 1.5.6        |
| os                                          | 0.9.6        |
| signet                                      | 0.7.3        |
| googleauth                                  | 0.5.3        |
| faraday_middleware                          | 0.12.2       |
| http-cookie                                 | 1.0.3        |
| unf_ext                                     | 0.0.7.2      |
| unf                                         | 0.1.4        |
| domain_name                                 | 0.5.20170404 |
| claide                                      | 1.0.2        |
| colored2                                    | 3.1.2        |
| nanaimo                                     | 0.2.3        |
| xcodeproj                                   | 1.5.2        |
| bundler                                     | 1.16.0.pre.2 |
| unicode-display_width                       | 1.3.0        |
| fastlane-plugin-automated_test_emulator_run | 1.5.1        |
| fastlane-plugin-changelog                   | 0.7.0        |
| fastlane-plugin-demo_mode                   | 1.0.1        |
```

**Comment by mathroule** [comment]
On Android 26, Pixel Launcher do not crash after changing locale.

**Comment by dwhitlow** [comment]
I'm experiencing the same issue. I can consistently reproduce this on an AVD with any hardware profile using the x86 API Level 25 system image (with Google APIs; I don't see an option to create an AVD without them for level 25). I also tried a variety of hardware profiles with x86 API Level 24 (with and without Google APIs) and could not reproduce the issue on any of them.

I attempted to reproduce this on the armeabi-v7a API Level 25 image as well, but gave up after waiting 5-10 minutes for screengrab to install the APK.
